src/RpcClient.ts

```diff
@@ -279,17 +285,84 @@ export class RedirectRpcClient extends RpcClient {
     /* tslint:disable-next-line:no-empty */
     public close() { }
 
-    public call(returnURL: string, command: string, handleHistoryBack = false, ...args: any[]) {
-        this.callAndSaveLocalState(returnURL, null, command, handleHistoryBack, ...args);
+    public call(
+        returnURL: string,
+        command: string,
+        optionsOrHandleHistory?: CallOptions | boolean | null,
+        ...args: any[]): void {
+        if (!optionsOrHandleHistory || typeof optionsOrHandleHistory === 'boolean') {
+            console.warn('RedirectRpcClient.call(string, string, boolean, any[]) is deprecated.'
+                + ' Use RedirectRpcClient.call(string, string, CallOptions, any[]) with an'
+                + ' appropriate CallOptions object instead.');
```

Comment:
Is `!optionOrHandleHistory` considered a deprecated function call? I.e. is `CallOptions` now always required even if I just want to use the defaults?
I think `!optionOrHandleHistory` could still be allowed and only be warned on `typeof optionOrHandleHistory === 'boolean'`

Reply:
Agreed. Omitting the `optionOrHandleHistory` parameter is valid, even in the new call signature.

Reply:
Of course, you are correct.